src/transaction/log_global.c

```diff
@@ -32,6 +32,8 @@
 #include "mvcc_table.hpp"
 #include "porting.h"
 #include "storage_common.h"
+#include "server_type.hpp"
```

Comment:
order of includes - alphabetical (as far as I know)

Reply:
onde